### ISSUE TITLE
Support multiple zones in one certificate

### ISSUE THREAD

**Issue by prices** [aaomidi/certbot-dns-google-domains#32]
It would be nice if multiple zone credentials could be put into one call so that a certificate with multiple zones in it could be created or renewed.

I have certificates that include domains from more than one zone.

Thanks!

**Comment by aaomidi** [comment]
I think the access token is per zone, right? Not sure if it’s a trivial thing to support at the plug-in level.  

I’ll look into it a bit.

**Comment by prices** [comment]
Yes, the access token is per zone.  I can also dig into it a little bit, but truthfully I am terrible at python.

One thought I did have is if you could support multiple --dns-google-domains-credentials arguments that could be a way to get multiple credentials in.

**Comment by vg** [comment]
If [this](https://github.com/certbot/certbot/pull/6644) ever gets merged we will be able to use just one access token. Till then not possible.

**Comment by aaomidi** [comment]
I'll be honest, there's way better software than certbot to do ACME at this point.

**Comment by earithramir** [comment]
Any thoughts on this? Or maybe a way for configuring Googles permission to do the same?
I was looking into this too, as my mail server serves multiple domains but only supports a single domain using multiple SAN (for other domain names). And the Google Domains ACME Api keys can (at least as far as i know) only control the DNS for a single domain name.

**Comment by prices** [comment]
I wrote a hook for dehydrated that does multiple domains on one certificate.

https://github.com/prices/dehydrated_hooks

I don't know enough about certbot to be able to modify this script.  I also have a lot of trouble with python.

**Comment by earithramir** [comment]
> I wrote a hook for dehydrated that does multiple domains on one certificate.
> 
> https://github.com/prices/dehydrated_hooks
> 
> I don't know enough about certbot to be able to modify this script. I also have a lot of trouble with python.

Thanks I'll review the code and modify it, if needed, keep u posted!
Although I haven't done much in python I'll manage, I think :)
